docs/2_2_common_mechanisms.adoc

```diff
@@ -330,522 +343,711 @@ Set <<parameter,`parameters`>>, <<input,`inputs`>>, and <<start>> values, and re
+
+|====
+
+[[time-based-clocks,time-based clocks]]Time-based clocks::
+
+are in a sense predictable and help the importer to take these clock ticks into account a priori.
+All time-based clocks are defined to be input clocks because the importer calls <<fmi3SetClock>> or <<fmi3ActivateModelPartition>> on these clocks.
+The importer queries the FMU about when a time-based clock should be activated.
+The importer will then activate the clock by calling <<fmi3SetClock>> with `values == fmi3ClockActive`.
++
+The mathematical descriptions of <<time-based-clocks>> will use the following notations:
 
-_[Outside of the FMU it is not known in advance when this <<clock>> ticks._
-_Instead, only when the <<clock>> is activated by the FMU, then the environment is informed that the <<clock>> ticks at this time instant._
-_It is the task of the environment to handle the messaged events appropriately based on the <<clockReference>> information._
-_Example:_
-// TODO: add example
-_]_
++
+.Mathematical Notation Description.
+[#table-notation-description]
+[cols="1,2"]
+|===
+|latexmath:[\mathbf{t}_0]
+|The time instant at which the <<clock>> is activated the first time.
 
+|latexmath:[\mathbf{t}_\mathit{i-1}]
+|The previous time instant, where the <<clock>> ticked.
 
-_[TODO: local clocks should be introduced here too, they are just used afterwards.
-Issue 1172 will address the definition of local variables (including local clocks).
-Then we can introduce a section here on local clocks.
-]_
+|latexmath:[\mathbf{T}_{\mathit{shift}}]
+|Delay for the first clock activation relative to latexmath:[\mathbf{t}_{\mathit{start}}], defined differently for the different clock types.
+latexmath:[\mathbf{T}_{\mathit{shift}}] can be retrieved from the FMU with <<fmi3GetShiftDecimal>> as floating point value, or with <<fmi3GetShiftFraction>> as rational number: latexmath:[\mathit{shiftCounter} / \mathit{resolution}] (see <<resolution>>).
 
-===== Input Clocks [[inputClockVariant]]
+|latexmath:[\mathbf{T}_{interval, i}]
+|The time interval until the next <<clock>> tick, defined differently for the different clock types.
+latexmath:[\mathbf{T}_{interval, i}] can be set with <<fmi3SetIntervalDecimal>>) or get with <<fmi3GetIntervalDecimal>> as floating point value, or as rational number using <<fmi3SetIntervalFraction>> or <<fmi3GetIntervalFraction>>: latexmath:[\mathit{intervalCounter} / \mathit{resolution}] (see <<resolution>>).
 
-An <<inputClock>> is a <<periodic>> or <<periodic,aperiodic>> <<clock>> that is defined by a <<clock>> outside of the FMU.
-I.e., only when the <<clock>> is activated by the environment of the FMU, then the FMU is informed that the <<clock>> ticks at this time instant.
-An <<inputClock,input clocks>> must be independent to all other input clocks.
-_[TODO: what does that mean?]
-If dependent clocks exist, they can be exposed as <<local>> or <<output>> clocks.
+|latexmath:[\mathbf{t}_\mathit{event}]
```

Comment:
Why not \mathbf{t}_\mathit{event, i} since this event occurs several times. It looks strange without a substcript i in equations with t_i and T_interval,i, don't you think?

Reply:
It would have to be a different subscript j because there is no synchronization of event numbers and ticks of that clock.
Therefore, this is likely ok. However, t_event does not fit when we talk about SE - and SE is listed... here, this needs clarification for the definition of t_event. I made a proposal in the PR.
Please <<resolve conversation>> if you are happy enough with that proposal.

Reply:
Changed tactics: I close, you reopen if unhappy (I am drowning in change requests, sorry.)